CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Allow matching compounded subcategories from SV callers e.g. DUP:INV (#5360)
 - Adjust the link to the chanjo2 gene coverage report to reflect the type of analyses used for the samples
 - Gene panels open in new tabs from case panels and display case name on the top of the page
+- When uploading research variants, use rank threshold defined in case settings, if available, otherwise use a default threshold of 8
```

Comment:
```suggestion
- When uploading research variants, use rank threshold defined in case settings, if available, otherwise use the default threshold of 8
```